### PR TITLE
feat: move fast_track to master

### PR DESCRIPTION
**Pull request by hkad98**
---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18709/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20401/ : FAILURE in 1m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18711/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20405/ : FAILURE in 1m 07s

File: examples/playground/src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // (C) 2019-2024 GoodData Corporation
-import React, { useMemo } from "react";
```
Comment (ivanmjartan) [inline examples/playground/src/App.tsx:2]: remove all changes from playground

File: examples/sdk-interactive-examples/scripts/validateExamplesDependencies.mjs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import fs from "fs";
+import path from "path";
```
Comment (ivanmjartan) [inline examples/sdk-interactive-examples/scripts/validateExamplesDependencies.mjs:2]: remove all changes from examples

File: libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/loadDashboardAlerts.ts [hunk 3]
```diff
@@ -1,14 +1,15 @@
-// (C) 2021-2022 GoodData Corporation
+// (C) 2021-2024 GoodData Corporation
 import { IWidgetAlert } from "@gooddata/sdk-model";
 import { DashboardContext } from "../../../types/commonTypes.js";
 
 export function loadDashboardAlerts(ctx: DashboardContext): Promise<IWidgetAlert[]> {
     const { backend, workspace, dashboardRef } = ctx;
 
+    // SKIPPING THIS FOR FAST TRACK
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/loadDashboardAlerts.ts:8]: fix this

File: libs/sdk-ui-dashboard/src/model/store/alerts/alertsSelectors.ts [hunk 4]
```diff
@@ -35,32 +45,54 @@ interface IWidgetAlertMapItem {
...
-            if (!widget) {
-                /**
-                 * Ignore widgets that are no longer available, this can naturally happen in edit mode when
-                 * a KPI widget is deleted by the user: the alerts are not removed from redux in case the edit
-                 * mode is cancelled (to avoid having to load the dashboard again from the backend).
-                 */
-                return undefined;
-            }
+const selectAlertsMapByWidgetRefs = createSelector(
+    selectAlerts,
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/src/model/store/alerts/alertsSelectors.ts:49]: nOt sure about this fix SDK only support tiger so no alerts ... if this fix necessary for bear than we need to propagate to different branch

File: libs/sdk-ui-dashboard/src/model/store/config/configSelectors.ts [hunk 5]
```diff
@@ -570,6 +570,18 @@ export const selectIsDrillDownEnabled: DashboardSelector<boolean> = createSelect
     },
 );
 
+/**
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/src/model/store/config/configSelectors.ts:573]: Not sure about this selector ... I guess we have already one defined

File: libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextSelectors.ts [hunk 6]
```diff
@@ -455,7 +455,7 @@ export const selectIsCircularDependency: (
         ),
 );
 
-const MAX_DRAGGABLE_FILTERS_COUNT = 30;
+const MAX_DRAGGABLE_FILTERS_COUNT = 1_000;
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextSelectors.ts:458]: I do not understand this value ?

File: libs/sdk-ui-dashboard/src/presentation/widget/widget/InsightWidget/InsightWidgetAlert.tsx [hunk 7]
```diff
@@ -0,0 +1,312 @@
+// (C) 2023-2024 GoodData Corporation
+import React, { useCallback, useMemo, useState } from "react";
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/src/presentation/widget/widget/InsightWidget/InsightWidgetAlert.tsx:2]: Not sure about support alerts and tiger

File: libs/sdk-ui-dashboard/styles/scss/fast_track.scss [hunk 8]
```diff
@@ -0,0 +1,123 @@
+// (C) 2024 GoodData Corporation
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/styles/scss/fast_track.scss:1]: rename this css or merge it

File: libs/sdk-ui-ext/src/internal/constants/uiConfig.ts [hunk 9]
```diff
@@ -4,22 +4,22 @@ import { IUiConfig } from "../interfaces/Visualization.js";
...
-export const INCREASE_MAX_TABLE_MEASURE_ITEMS_LIMIT = 100;
-export const INCREASE_MAX_TABLE_ATTRIBUTES_ITEMS_LIMIT = 50;
-export const MAX_STACKS_COUNT = 1;
-export const MAX_VIEW_COUNT = 2;
-export const DEFAULT_HEADLINE_METRICS_COUNT = 1;
-export const DEFAULT_XIRR_METRICS_COUNT = 1;
-export const DEFAULT_XIRR_ATTRIBUTES_COUNT = 1;
-const DEFAULT_GEO_ATTRIBUTES_COUNT = 1;
-const DEFAULT_PUSHPIN_METRICS_COUNT = 1;
+export const MAX_METRICS_COUNT = 1_000;
```
Comment (ivanmjartan) [inline libs/sdk-ui-ext/src/internal/constants/uiConfig.ts:7]: Change const to valid number 1000

File: libs/sdk-ui-ext/src/internal/constants/uiConfig.ts [hunk 10]
```diff
@@ -4,22 +4,22 @@ import { IUiConfig } from "../interfaces/Visualization.js";
...
+export const DEFAULT_PIE_METRICS_COUNT = 1_000;
```
Comment (ivanmjartan) [inline libs/sdk-ui-ext/src/internal/constants/uiConfig.ts:8]: change all const devinition to validn number not shure about ecma script 2021
Pls change it export const DEFAULT_PIE_METRICS_COUNT = 1000;

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18743/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20430/ : FAILURE in 10m 13s

File: libs/sdk-backend-tiger/api/sdk-backend-tiger.api.md [hunk 11]
```diff
@@ -453,6 +483,25 @@ export type TigerSpecificFunctions = {
     analyzeCsv?: (dataSourceId: string, analyzeCsvRequest: AnalyzeCsvRequest) => Promise<Array<AnalyzeCsvResponse>>;
     importCsv?: (dataSourceId: string, importCsvRequest: ImportCsvRequest) => Promise<void>;
     listFiles?: (dataSourceId: string) => Promise<Array<GdStorageFile>>;
+    computeKeyDrivers?: (workspaceId: string, metric: IMeasure, sortDirection: "ASC" | "DESC") => Promise<IKeyDriverAnalysis>;
```
Comment (kandl) [inline libs/sdk-backend-tiger/api/sdk-backend-tiger.api.md:486]: All these method should be likely moved and supported by `sdk-backend-spi`.

**Review by kandl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**